auth/auth-route.js

```diff
@@ -0,0 +1,55 @@
+const router = require( 'express' ).Router();
+const bcrypt = require( 'bcryptjs' );
+const session = require( '../api/server' );
```

Comment:
I don't think you need to import anything from server here. The session library adds itself to the req object so you should have access without this I believe